code/src/Eduweb/TrainingBundle/Controller/AdminController.php

```diff
@@ -0,0 +1,111 @@
+<?php
+
+namespace Eduweb\TrainingBundle\Controller;
+
+use Eduweb\TrainingBundle\Entity\Contact;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Template;
+use Symfony\Component\HttpFoundation\Request;
+use Eduweb\TrainingBundle\Form\Type\RegisterType;
+
+/**
+ * Class BlogController
+ *
+ * @package Eduweb\TrainingBundle\Controller
+ * @Route("/blog/admin")
+ *
+ */
+class AdminController extends Controller
+{
+    /**
+     * @Route("/", name="edu_blog_admin_listing")
+     *
+     * @Template
+     */
+    public function listingAction()
+    {
+        $repo = $this->getDoctrine()->getRepository('EduwebTrainingBundle:Register');
+        $rows = $repo->findBy(array('sex' => ''));
```

Comment:
I was testing some possibilities. I check, that, Can I find item with empty string in database. This is only test, I deleted this know because it is not necessary.